docs/time/index.rst

```diff
@@ -284,7 +285,10 @@ appropriate::
    [ 50002.   50002.5]]>
   >>> t[0]
   <Time object: scale='utc' format='mjd' value=[ 50000.   50000.5]>
-
```

Comment:
Per your question about location of these docs, I think it would be good to just make a new section here entitled `Numpy method analogs`.  In this section explicitly list all the numpy-like methods that are supported.  Discuss when memory is shared or not (i.e. just say that it is the same as Numpy, but some readers won't necessarily know what that means offhand).  I don't think of `take` as a "shape-changing" method as much as an item selection method, but anyway calling these "Numpy analogs" seems more clear to me.